lib/guard/plugin_util.rb

```diff
@@ -55,7 +55,7 @@ def initialize(name)
     def initialize_plugin(options)
       klass = plugin_class
       fail "Could not load class: #{_constant_name.inspect}" unless klass
-      if klass.superclass.to_s == "Guard::Guard"
```

Comment:
@ioquatix This is to support legacy Guard plugins that inherits from `Guard::Guard`, I don't see how `klass.superclass.eql?(Guard)` or `klass.ancestors.include?(Guard)` improves the situation if you're plugin is a module?